backend/tests/test_project.py

```diff
@@ -298,3 +298,58 @@ def test_create_project(self):
         self.assertEqual(empty_response.status_code, 400)
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response2.status_code, 200)
+
+    def test_get_contacts_by_project(self):
+
+        c0_id = uuid.uuid4()
+        c0 = Contact(
+            id=c0_id,
+            name="dummy name",
+            email="dummy email",
+            cellphone="dummy cellphone",
+            role="dummy role",
+            organization="dummy organization",
+            neighbourhood="dummy neighbourhood",
```

Comment:
`    contact_type = db.Column(
        UUID(as_uuid=True), db.ForeignKey("contact_types.id"), nullable=False
    )`
the contact_type field in Contacts are nullable=false, this should have possibly failed ?

Reply:
@peterghrong It won't fail as I've attached the two contacts to the contact type below, and because we've declared a one-to-many relationship, it will automatically update when inserting into the database. `nullable=False` only raises an exception during the insertion part and not the creation part. As seen from the test logs in travis, it passes.

Reply:
Thanks for the explanation, LGTM then !

Reply:
@peterghrong I await the approval, lmk if you have any questions.

Reply:
Naww cannot think of anything else champ, forgot to approve it soz